runtime/oti/vm_api.h

```diff
@@ -558,7 +558,7 @@ internalExceptionDescribe(J9VMThread *vmThread);
 * @return UDATA
 */
 UDATA
-iterateStackTrace(J9VMThread * vmThread, j9object_t* exception,  UDATA  (*callback) (J9VMThread * vmThread, void * userData, J9ROMClass * romClass, J9ROMMethod * romMethod, J9UTF8 * fileName, UDATA lineNumber, J9ClassLoader* classLoader, J9Class* ramClass), void * userData, UDATA pruneConstructors);
+iterateStackTrace(J9VMThread * vmThread, j9object_t* exception,  UDATA  (*callback) (J9VMThread * vmThread, void * userData, UDATA bytecodeOffset, J9ROMClass * romClass, J9ROMMethod * romMethod, J9UTF8 * fileName, UDATA lineNumber, J9ClassLoader* classLoader, J9Class* ramClass), void * userData, UDATA pruneConstructors);
```

Comment:
Can you add the new parameter to the `@param` list above?

Reply:
Oops, missed this, will add it.

Reply:
Renamed `method` to `romMethod`, added `bytecodeOffset`, `romClass`, and `classLoader`.